tests/integration/workflows/test_workflows_tasks_actions.py

```diff
@@ -904,7 +904,7 @@ def test_normalize_collaborations(workflow_app, insert_experiments_into_db):
     assert obj.data['accelerator_experiments'] == expected_accelerator_experiments
 
 
-def test_normalize_collaborations_one_collaboration(workflow_app, insert_experiments_into_db):
+def test_normalize_collaborations_no_collaboration(workflow_app, insert_experiments_into_db):
```

Comment:
why not keep both tests?

Reply:
I added this test in the previous pr. But it doesn't make sense now. We need to test multiple collaborations and 0 collaboration. No need for testing a single collaboration.